build.rs

```diff
@@ -167,3 +168,9 @@ fn get_os_process() -> String {
         String::from("bash")
     }
 }
+
+fn current_dir() -> String {
+    std::env::current_dir()
+        .map(|d| d.display().to_string())
+        .unwrap_or_else(|_| String::from("<CURRENT DIR>"))
```

Comment:
Given the usage, maybe `<COULDN'T GET CURRENT DIR>`?  To a reader of test failure messages, it might be a bit clearer?

Reply:
I don't think so. The current directory will be used. If we say "<COULDN'T GET CURRENT DIR>" the user has no idea what directory. If the say "<CURRENT DIR>" maybe they'll understand that it means whatever the current directory is.